charts/kubefed/charts/controllermanager/templates/clusterrolebindings.yaml

```diff
@@ -17,7 +17,14 @@ subjects:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: auth-delegator-kubefed-admission-webhook
+# For namespace scoped deployments, create a unique cluster-scoped resource
```

Comment:
(No action required) Given that this comment applies for each of the 3 conditionals that appear in this file, consider referencing it in the other 2 rather than duplicating the comment.

Reply:
I felt it was cleaner to just copy/paste the comment since it's a manifest file that shouldn't really change instead of providing references across files and lines where context may be lost. Note that there are only 2 conditionals in this file where only part of this message is duplicated.

Reply:
As per my comment above, the danger of copy-pasting large blocks of duplicative comments is that they end up not being read exhaustively.  I would prefer a bias towards relevant rather than exhaustive commentary.